lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,13 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    punctuation = '!@#$%^&*()-_+=[{]};:\'"<>/?,.\\|/'
+    text_no_punctuation = ''.join(char for char in text if not char not in punctuation)
```

Comment:
`not char not in punctuation` is equal to `char in punctuation`, right? is that what you intended to do?

Reply:
fixed

Reply:
is it?